components/filter/filter.js

```diff
@@ -125,12 +154,13 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		});
 	}
 
-	_buildHeader() {
```

Comment:
Most of the changes in this function are setting things up for future changes, where there will be a clear button, search bar, etc.